code/datums/outfits/outfit_admin.dm

```diff
@@ -256,6 +256,11 @@
 	if(visualsOnly)
 		return
 
+	var/obj/item/weapon/rig/light/ninja/rig = new /obj/item/weapon/rig/light/ninja(src)
+	if(H.dna)
+		rig.dnaLock = H.dna
+	H.equip_to_slot_or_del(rig, slot_l_hand)
```

Comment:
Why not back?

Reply:
Equipping it to back causes a timer unless some gimmicky code is used, which stalls the start of the game and causes people to spawn in space. This also allows the ninja to simply put the hardsuit down and modify it with modules bought from the store.